app/bootstrap.php

```diff
@@ -1,17 +1,3 @@
 <?php
 
-use WMDE\Fundraising\Deployment\ReleaseStateWriter;
-
-// Initialize services
-
-$app = new Silex\Application();
-
-$app['db'] = function ( $app ) {
-	return new \PDO( $app['dsn'] );
-};
-
-$app['release_state_writer'] = function ( $app ) {
-	return new ReleaseStateWriter( $app['db'] );
-};
-
-return $app;
\ No newline at end of file
+return new Silex\Application();
```

Comment:
Until we do some real bootstrapping (which won't happen now that we've got a TopLevelfactory) we can get rid of this file and just create `$app` in `index.php`